pyiron_base/state/settings.py

```diff
@@ -284,12 +284,12 @@ def _validate_viewer_configuration(config: Dict) -> None:
     @staticmethod
     def _validate_no_database_configuration(config: Dict) -> None:
         if "disable_database" in config.keys() and config["disable_database"]:
-            if config["project_check_enabled"]:
+            if "project_check_enabled" in config.keys() and config["project_check_enabled"]:
```

Comment:
I am a bit surprised to see this in this PR, but the changes are ok to me. Did they pop up here or just sneak in?

Reply:
They sneaked in